kuzzleio/kuzzle-documentation-cli
# Beta

## What does this PR do ?

This PR adds many things:

* Adds semantic-release
* Adds linting command
* Lint the project
* Change commands that used $(npm bin) since it is not available anymore in npm@9



### How should this be manually tested?


  - Step 1 : npm install -g kuzdoc@2.6.0-beta.2 as it has been deployed as a pre-release
  - Step 2 : Enter the documentation folder and launch kuzdoc test command to start the documentation with a repo
  - Step 3 : Go to localhost:8080 and see your repo

### Other changes



### Boyscout
